bugs/z3/nra-1.smt2

```diff
@@ -0,0 +1,25 @@
+; config-1
```

Comment:
I would also add a line with a link to the issue reporting this, so we know what was reported and we can also easily track them, otherwise a better solution would be to add a top level .md file with internal reference to the bug files and a link there to the issue, what do you think?

Reply:
In the second case you can also leave it to me :)

Reply:
I'll add a markdown file with a table

Reply:
Thanks!